TYPO3.TypoScript/Classes/TYPO3/TypoScript/Core/ExceptionHandlers/AbstractRenderingExceptionHandler.php

```diff
@@ -70,7 +71,7 @@ public function handleRenderingException($typoScriptPath, \Exception $exception)
         if ($this->exceptionDisablesCache($typoScriptPath, $exception)) {
             $this->runtime->setEnableContentCache(false);
         }
-        $referenceCode = ($exception instanceof \TYPO3\Flow\Exception) ? $exception->getReferenceCode() : null;
+        $referenceCode = ($exception instanceof Exception) ? $exception->getReferenceCode() : null;
```

Comment:
In case of Exceptions named "Exception" I would prefer to add some context. Like `Flow\Exception`in this case.